### PR TITLE
azuredatastudio: 1.17.1 > 1.26.0

### PR DESCRIPTION
**Pull request by pietdevries94**
###### Motivation for this change
The package hadn't had an update in a long while.
I added the new dependencies and added the .desktop files from the Debian package in the same way as the VSCode package, where Azure Data Studio is a fork of.

I made x86_64-linux the only platform, because this package is clearly broken on anything else. In a future pull request, we should migrate the VSCode update scripts to Azure Data Studio.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112139` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>azuredatastudio</li>
</ul>
</details>

File: pkgs/applications/misc/azuredatastudio/default.nix [hunk 1]
```diff
@@ -104,12 +148,20 @@ stdenv.mkDerivation rec {
       ${targetPath}/bin/${edition} \
       $out/bin/azuredatastudio \
       --set LD_LIBRARY_PATH ${rpath}
+
+    mkdir -p $out/share/applications
+    ln -s $desktopItem/share/applications/azuredatastudio.desktop $out/share/applications/azuredatastudio.desktop
+    ln -s $urlHandlerDesktopItem/share/applications/azuredatastudio-url-handler.desktop $out/share/applications/azuredatastudio-url-handler.desktop
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/azuredatastudio/default.nix:154]: Please use copyDesktopItems instead.

**Comment by pietdevries94** [comment]
Updated the package from 1.25.3 > 1.26.0

File: pkgs/applications/misc/azuredatastudio/default.nix [hunk 2]
```diff
@@ -38,7 +84,7 @@ stdenv.mkDerivation rec {
     at-spi2-atk
   ];
 
-  phases = "unpackPhase fixupPhase";
+  phases = "unpackPhase installPhase";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/azuredatastudio/default.nix:87]: Please do not overwrite phases and disable the ones you do not need.

**Comment by avdv** [comment]
Hi.

@pietdevries94   Any updates on this? Current version is 1.32.0...

**Pull request closed without merging**